source/frontend/src/components/common/Section/Section.tsx

```diff
@@ -9,18 +9,29 @@ interface SectionProps {
   header?: React.ReactNode;
   isCollapsable?: boolean;
   initiallyExpanded?: boolean;
+  noPadding?: boolean;
 }
 
 export const Section: React.FC<
   React.PropsWithChildren<SectionProps & React.HTMLAttributes<HTMLDivElement>> & {
     'data-testid'?: string;
   }
-> = ({ header, children, title, isCollapsable, initiallyExpanded, className, ...rest }) => {
+> = ({
+  header,
+  children,
+  title,
+  isCollapsable,
+  initiallyExpanded,
+  noPadding,
+  className,
+  ...rest
+}) => {
```

Comment:
These (and the modal changes) are in your other PR. They don't look related to this pr. Please make sure to include changes relevant to document model refactoring - thanks!

Reply:
these changes are actually related to the document UI changes. Once that pr is merged these will not show on the diff